torchmetrics/functional/text/bert.py

```diff
@@ -449,9 +451,16 @@ def _rescale_metrics_with_baseline(
     return all_metrics[..., 0], all_metrics[..., 1], all_metrics[..., 2]
 
 
+@deprecated(
+    args_mapping={"predictions": "preds", "references": "target"},
+    target=True,
+    stream=partial(warn, category=FutureWarning),
```

Comment:
lets remove this argument for now as it shall be done eventually consistently for all, so we can make it as separate PR